src/ints/bios.cpp

```diff
@@ -1446,6 +1482,11 @@ class BIOS final : public Module_base{
 	}
 };
 
+void BIOS::AddMessages()
+{
+	MSG_Add("BIOS_REBOOT_REQUESTED", "Reboot requested...");
```

Comment:
Printing out the following would be cool, but not worth the complication if it's too difficult.

```
Rebooting in 3 seconds...
```

Then decrement 3 to 2, then to 1, and then reboot (with 1 second pauses).

Reply:
Not difficult at all. Changed.